content/updates/2021-03/1-Asset-mass-edit.md

```diff
@@ -0,0 +1,19 @@
+---
+pim_announcement_alt_img: "Asset mass edit screen"
+pim_announcement_audience:
+- EE
+---
+
+# Mass edit your assets
+::: meta-data type="New" features="Productivity" available="late march" in="EE" link-to-doc="../articles/TODO"
+
+If you need to edit several assets from the same family at once, it's now possible to do it in the PIM user interface. No doubt it will save you a tremendous amount of time!
+
+![Asset mass edit screen](../img/TODO.png)
+
+This new mass edition feature is all the more powerful since you can choose several asset attributes to modify at once. Neat!
```

Comment:
I'd remove this sentence, it's induced in my suggestion above. It's important information, it should be mentioned from the beginning.

Reply:
```suggestion
```